app/views/api/integrations/discord/ping.jbuilder

```diff
@@ -0,0 +1,3 @@
+return unless @resolver.response_status == Discord::Interactions::Resolvers::Ping::RESPONSE_STATUS[:OK]
```

Comment:
this is not a responsability that belongs to the serializer, it should belong to the controller

Reply:
Done!